proto/prysm/v1alpha1/beacon_block.proto

```diff
@@ -1054,24 +1042,6 @@ message BlindedBeaconBlockBodyElectra {
   ethereum.engine.v1.ExecutionRequests execution_requests = 13;
 }
 
-message AttesterSlashingElectra {
```

Comment:
should this be going into core? if we go this way wouldn't it make more sense to have it in its own electra proto?